kubetest/gke.go

```diff
@@ -336,6 +336,7 @@ func (g *gkeDeployer) Up() error {
 		// gcloud enables node auto-upgrade by default, which doesn't work with CUSTOM image.
 		// We disable auto-upgrade explicitly here.
 		args = append(args, "--no-enable-autoupgrade")
+		args = append(args, "--no-enable-shielded-nodes")
```

Comment:
Commit message is good for this line if I look at the history in the future.

Might want to add to the comment above or create a separate inline comment with the reason.

Reply:
Done